noxfile.py

```diff
@@ -370,6 +370,7 @@ def benchmarks(
     COMPARE_FACTOR = 2.0
 
     session.install("asv", "nox", "pyyaml")
+    session.run("conda", "install", "--yes", "conda<24.3")
```

Comment:
@stephenworsley Consider making an issue to remind ourselves to unpin this whenever `asv` updates to not use the `--force` flag 👍